pycode/memilio-epidata/setup.py

```diff
@@ -61,7 +61,8 @@ def run(self):
             'getsimdata = memilio.epidata.getSimulationData:main',
             'cleandata = memilio.epidata.cleanData:main',
             'getcasesestimation = memilio.epidata.getCaseDatawithEstimations:main',
-            'getcommutermobility = memilio.epidata.getCommuterMobility:main'
+            'getcommutermobility = memilio.epidata.getCommuterMobility:main',
+            'getvaccinationdata = memilio.epidata.getVaccinationData:main'
```

Comment:
Is getcasesestimation missing?

Reply:
No. Here getcasesestimation was already added before.